Lib/test/test_builtin.py

```diff
@@ -358,6 +361,32 @@ def f(): """doc"""
                 rv = ns['f']()
                 self.assertEqual(rv, tuple(expected))
 
+    def test_compile_top_level_await(self):
+        for mode in ('single', 'exec'):
+            for num,code_sample in enumerate(['''await sleep(0)''',
```

Comment:
Minor nit : num is not used in test so `enumerate` seems to be not needed . Perhaps the list used in enumerate can be defined as a variable outside of for loop and just iterated?

Reply:
Good point, I used it to debug which sample(s) were wrong and did not removed it, will do.